src/bit-fields/src/example.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+extern crate self as bit_fields;
+bit_fields::bitfield!(ExampleBitField,u32,{
```

Comment:
I can't find this type (`ExampleBitField`) used anywhere? If it was just intended for demonstrating usage of the `bitfield!` macro, but isn't executed anywhere, I think the file should be removed.

Reply:
I think it is useful to show usage and the generated functionality, `bitflags` does the same with https://docs.rs/bitflags/latest/bitflags/example_generated/struct.Flags.html.